test-core-1/upgrades/v7/proposal.md

```diff
@@ -0,0 +1,26 @@
+# Persistence v7
+
+This is a proposal to do a software upgrade to the v7.0.0-rc0 software tag of the Persistence codebase on block height 10209300, which is estimated to occur on **[10209300](https://testnet.mintscan.io/persistence-testnet/blocks/10209300)**. Block times have high variance, so please monitor
```

Comment:
`This is a proposal to do a software upgrade to the v7.0.0-rc0 software tag of the Persistence codebase on block height **[10209300](https://testnet.mintscan.io/persistence-testnet/blocks/10209300)**, which is estimated to occur on Wednesday 15 February 2023 around 2PM GMT. Block times have high variance, so please monitor`